app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,91 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+$light-gray: #777;
+
+body {
+    padding-top: 60px;
+}
+
+section {
+    overflow:  auto;
+}
+
+textarea {
+    resize: vertical;
+}
+
+.center {
+    text-align:  center;
+    h1 {
+        margin-bottom: 10px;
+    }
+}
+
+
+h1, h2, h3, h4, h5, h6 {
+    line-height: 1;
+  }
+  
+  h1 {
+    font-size: 3em;
+    letter-spacing: -2px;
+    margin-bottom: 30px;
+    text-align: center;
+  }
+  
+  h2 {
+    font-size: 1.2em;
+    letter-spacing: -1px;
+    margin-bottom: 30px;
+    text-align: center;
+    font-weight: normal;
+    color:$gray-light;
+  }
+  
+  p {
+    font-size: 1.1em;
+    line-height: 1.7em;
+  }
+
+  #logo {
+    float: left;
+    margin-right: 10px;
+    font-size: 1.7em;
+    color: #fff;
+    text-transform: uppercase;
+    letter-spacing: -1px;
+    padding-top: 9px;
+    font-weight: bold;
+    :hover {
+    color: #fff;
+    text-decoration: none;
+  }
+}
+  
+  /* footer */
+
+footer {
+    margin-top: 45px;
+    padding-top: 5px;
+    border-top: 1px solid #eaeaea;
+    color: $gray-light;
+    a {
+        color: #555;
+        &:hover {
+            color: #222;
+        }
+    }
+    small {
+        float: left;
+    }
+    ul {
+        float: right;
+        list-style: none;
+        li {
+            float:left;
+            margin-left: 15px;
+        }
+    }
+
+  }
```

footer のスタイルがどこまであるかわかるようにインデントも合わせたら良さそう

```suggestion
  margin-top: 45px;
  padding-top: 5px;
  border-top: 1px solid #eaeaea;
  color: $gray-light;
  a {
      color: #555;
      &:hover {
          color: #222;
      }
  }
  small {
      float: left;
  }
  ul {
      float: right;
      list-style: none;
      li {
          float:left;
          margin-left: 15px;
      }
  }
}
```